canonical/vanilla-framework
# New full-width layout for Design System website [WD-775]

## Done

Adds new layout components necessary to build new full-width site layout for Vanilla and Design System sites.

Fixes WD-775

This is currently considered experimental and internal feature, so no documentation is provided yet. We just want to release it and start using it on design.ubuntu.com and Vanilla later on.

## QA

- Open [demo](https://vanilla-framework-4621.demos.haus/docs/examples/layouts/full-width/default)
- Test the layout on various screen sizes
- Check the version without side navigation:
  - https://vanilla-framework-4621.demos.haus/docs/examples/layouts/full-width/no-sidebar
  - (you can navigate between them by clicking on logo or any top nav item)

### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
  - if CSS class names are not changed it can be bugfix relesase (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical/vanilla-framework/wiki/Release-process#pre-release-tasks)


## Screenshots

<img width="1512" alt="image" src="https://user-images.githubusercontent.com/83575/214291939-5da3d7f5-e06a-4cde-9b1c-acab9533f85e.png">
